config/ember-try.js

```diff
@@ -1,80 +1,76 @@
-'use strict';
+"use strict";
 
-const getChannelURL = require('ember-source-channel-url');
+const getChannelURL = require("ember-source-channel-url");
 
 module.exports = function() {
   return Promise.all([
-    getChannelURL('release'),
-    getChannelURL('beta'),
-    getChannelURL('canary')
-  ]).then((urls) => {
+    getChannelURL("release"),
```

Comment:
@lifeart were all these quotes changed to double quotes on purpose? I thought the blueprint used single quotes?

Reply:
@rwjblue it can be an `eslint` or `prettier` auto format rule

Reply:
We should use whatever the default blueprint format is. I think it is single quote in all the Ember repos.